### ISSUE TITLE
paper/code conflict: using minimum Q in policy gradient

### ISSUE THREAD

**Issue by jpreiss** [haarnoja/sac#14]
The Soft Actor-Critic paper ([arXiv v2](https://arxiv.org/abs/1801.01290) says, in the last paragraph on page 5:

> We then use the minimum of the Q-functions for the value gradient in Equation 6 and policy gradient in Equation 13

However, the [code in `sac/algos/sac.py`](https://github.com/haarnoja/sac/blob/fba571f81f961028f598b0385d80cb286d918776/sac/algos/sac.py#L313) uses only one of Q functions in the policy gradient loss. It does use the minimum in the value gradient loss.

Is there a reason for the discrepancy? Thanks!

**Comment by haarnoja** [comment]
Good catch! We actually tried both versions and did not find much difference between them. We'll fix the code in the next release.
